utils/create_xblock.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+# Function to insert entry in alphabetical order in setup.py under "xblock.v1"
+insert_in_alphabetical_order() {
+  local entry="$1"
+  local setup_file="$2"
+
+  awk -v new_entry="$entry" '
+  BEGIN { added = 0; indent = "" }
+  /entry_points/ { print; next }
+  /"xblock.v1"/ {
+    print;
+    getline
+    if (match($0, /^[ \t]+/)) {
+      indent = substr($0, RSTART, RLENGTH)
+    }
+    while (getline > 0) {
+      if ($0 ~ /^\s*\]/) {
+        if (added == 0) {
+          print indent new_entry ","
+          added = 1
+        }
+        print
+        next
+      }
+      if (added == 0 && $0 > indent new_entry) {
+        print indent new_entry ","
+        added = 1
+      }
+      print
+    }
+    next
+  }
+  { print }
+  ' "$setup_file" > temp_setup.py && mv temp_setup.py "$setup_file"
+}
+
+# Function to insert import before __version__ variable in __init__.py
+insert_import_before_version() {
+  local import_entry="$1"
+  local init_file="$2"
+
+  awk -v new_import="$import_entry" '
+  BEGIN { added = 0 }
+  {
+    if (added == 0 && /^__version__/) {
+      print new_import
+      print ""
+      added = 1
+    }
+    print
+  }
+  ' "$init_file" > temp_init.py && mv temp_init.py "$init_file"
+}
+
+# Prompt user for input
+read -p "Enter XBlock name e.g thumbs: " xblock_name
+read -p "Enter XBlock class e.g ThumbsXBlock: " xblock_class
+
+# Define paths and filenames
+base_dir="xblocks_contrib/$xblock_name"
+init_file="$base_dir/__init__.py"
+xblock_file="$base_dir/$xblock_name.py"
+tx_dir="$base_dir/.tx"
+static_dir="$base_dir/static"
+css_file="$static_dir/css/$xblock_name.css"
+js_file="$static_dir/js/src/$xblock_name.js"
+templates_dir="$base_dir/templates"
+html_file="$templates_dir/$xblock_name.html"
+setup_file="setup.py"
+main_init_file="xblocks_contrib/__init__.py"
+conf_locale_dir="$base_dir/conf/locale"
```

Comment:
We should think on it as it will change the expected config path required by the library

docs:
https://edx.readthedocs.io/projects/xblock/en/latest/xblock-tutorial/edx_platform/edx_lms.html#internationalization-support
https://github.com/openedx/i18n-tools
https://github.com/openedx/staff_graded-xblock?tab=readme-ov-file